doc/porting_guide.md

```diff
@@ -390,51 +414,57 @@ Let's assume the new platform's name is np2000. And an early advise: as a rule o
 
             make PRESET=np2000 XI_BSP_TLS=myTLSlibrary
 
-    - for wolfssl and mbedtls the BSP TLS implementations are available in files: *src/bsp/tls/wolfssl/xi_bsp_tls_wolfssl.c* and *src/bsp/tls/mbedtls/xi_bsp_tls_mbedtls.c*
+    - for wolfssl and mbedtls the BSP TLS implementations are available in files: `src/bsp/tls/wolfssl/xi_bsp_tls_wolfssl.c` and `src/bsp/tls/mbedtls/xi_bsp_tls_mbedtls.c`
     - if you chose a third one: *myTLSlibrary* then you have to write your own implementation
-        - create a file *src/bsp/tls/myTLSlibrary/xi_bsp_tls_myTLSlibrary.c* and implement all function declared in file *include/bsp/xi_bsp_tls.h*
+        - create a file `src/bsp/tls/myTLSlibrary/xi_bsp_tls_myTLSlibrary.c` and implement all functions declared in `include/bsp/xi_bsp_tls.h`
         - as samples you can follow the two existing implementations: wolfssl and mbedtls
-    - create file *make/mt-config/mt-tls-myTLSlibrary.mk* and fill in with content similar to *mt-tls-wolfssl.mk* or *mt-tls-mbedtls.mk*. This lets know the build system the include directoy, the binary directory, the static libraries to link against and config flags of the custom TLS library.
-    - you have to also provide a script *xively-client-c/src/import/tls/download_and_compile_myTLSlibrary.sh* which downloads the source of the custom TLS library and builds it. As a sample to follow look at the two already existing solutions: *download_and_compile_wolfssl.sh* and *download_and_compile_mbedtls.sh* in the same directory.
+    - create a file `make/mt-config/mt-tls-myTLSlibrary.mk` and fill in with content similar to `make/mt-config/mt-tls-wolfssl.mk` or `make/mt-config/mt-tls-mbedtls.mk`. This is where you would defined the include directory, the static library directory, the static library file to link against and the config flags of the custom TLS library.
+    - you might also want to provide a script `src/import/tls/download_and_compile_myTLSlibrary.sh` which downloads the source of the custom TLS library and builds it. As a sample to follow look at the two already existing solutions: `download_and_compile_wolfssl.sh`and `download_and_compile_mbedtls.sh` in that same directory.
```

Comment:
As a sample to follow look at the...
Isn't follow OR look enough?

Again you haven't touched this part, so sorry, just recognised this may be a confusing spot.